src/com/projectkorra/projectkorra/firebending/HeatControl.java

```diff
@@ -273,6 +274,17 @@ private ItemStack getCooked(final ItemStack is) {
 			case KELP:
 				cooked = new ItemStack(Material.DRIED_KELP);
 				break;
+			case CHORUS_FRUIT:
+				cooked = new ItemStack(Material.POPPED_CHORUS_FRUIT);
+				break;
+			case WHEAT:
+				cooked = new ItemStack(Material.BREAD);
+				break;
```

Comment:
Don't think we should do this, considering it takes 3 wheat per bread. Plus wheat isn't in the list of cookables above.

Reply:
Yeah thats because I removed it from my PR but forgot the switch case 😅